data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -196,3 +197,58 @@
 HOLC_GRADE_D_TRACT_20_PERCENT_FIELD: str = "Tract is >20% HOLC Grade D"
 HOLC_GRADE_D_TRACT_50_PERCENT_FIELD: str = "Tract is >50% HOLC Grade D"
 HOLC_GRADE_D_TRACT_75_PERCENT_FIELD: str = "Tract is >75% HOLC Grade D"
+
+
+# Climate Change
+FEMA_LOSS_RATE_LOW_INCOME = f"At or above the {PERCENTILE}th percentile for FEMA's expected loss rate and is low income"
```

Comment:
I'm fine with just doing these statically for now without the fancy method I proposed in https://github.com/usds/justice40-tool/issues/921#issuecomment-986100492. We can add that later. 

However, even if we do them statically, let's make sure they are all consistent. For instance, in your description of the fields for the three FEMA measures, you currently describe them as: 

* FEMA's expected loss rate
* agriculture
* building loss 

So each one has a different "format". You also have three different formats for the variable names: 

* FEMA_LOSS_RATE_LOW_INCOME (doesn't include "population" anywhere and should)
* AGRICULTURE_LOSS_RATE_LOW_INCOME
* EXPECTED_BUILDING_LOSS_RATE_LOW_INCOME (includes the word "expected", unlike others)

Also, by convention, notice that all the other field names end in `_FIELD`. 

Can you change to something like: 

```
EXPECTED_POPULATION_LOSS_RATE_LOW_INCOME_FIELD = f"At or above the {PERCENTILE}th percentile for expected population loss rate and is low income"
EXPECTED_AGRICULTURE_LOSS_RATE_LOW_INCOME_FIELD = f"At or above the {PERCENTILE}th percentile for expected agriculture loss rate and is low income"
EXPECTED_BUILDING_LOSS_RATE_LOW_INCOME_FIELD = f"At or above the {PERCENTILE}th percentile for expected building loss rate and is low income"
``` 

As you'll notice we're fairly obsessed with consistency and standardization around here.

Thanks!!

Reply:
Thanks @lucasmbrown-usds  - I will try to be more consistent from here on out.